src/pages/ProfilePage.js

```diff
@@ -133,7 +133,8 @@ function ProfilePage(props) {
 
     const navigateBackTo = lodashGet(props.route, 'params.backTo', ROUTES.HOME);
 
-    const notificationPreference = !_.isEmpty(props.report) ? props.translate(`notificationPreferencesPage.notificationPreferences.${props.report.notificationPreference}`) : '';
+    const shouldShowNotificationPreference = !_.isEmpty(props.report) && props.report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN;
+    const notificationPreference = shouldShowNotificationPreference ? props.translate(`notificationPreferencesPage.notificationPreferences.${props.report.notificationPreference}`) : '';
```

Comment:
It looks like a redundant condition here. The only place `notificationPreference` is used is in the `title={notificationPreference}` and the only way that will show is if `shouldShowNotificationPreference` is `true` anyway. So, defaulting to an empty string is not needed.

Reply:
`hidden` doesn't currently have a translation since it doesn't show up anywhere on the app. So I wanted to add teh conditional so that props.translate doesn't have a warning since `hidden` doesn't exist.

Reply:
Hmm ok I think I see what you are saying now. But what does it show instead? Do we need some fallback?